source/WebConfig/cosa_webconfig_dml.c

```diff
@@ -100,6 +100,12 @@ X_RDK_WebConfig_SetParamBoolValue
 	return FALSE;
 }
 
+BOOL Get_RfcEnable()
```

Comment:
This func is not needed, direct struct param call is fine. This func call will be overhead